connectors/cli.py

```diff
@@ -37,8 +37,20 @@ def _parser():
     parser.add_argument(
         "--action",
         type=str,
-        default=["schedule", "execute", "cleanup"],
-        choices=["schedule", "execute", "list", "config", "cleanup"],
+        default=[
+            "schedule",
+            "execute_content_sync_job",
+            "execute_access_control_sync_job",
+            "cleanup",
+        ],
+        choices=[
+            "schedule",
+            "execute_content_sync_job",
+            "execute_access_control_sync_job",
```

Comment:
IMO it's too verbose.

We have `schedule`, this one could be `sync_content`, `sync_acs`?

Reply:
Renamed to `sync_content` and `sync_access_control`: https://github.com/elastic/connectors/pull/1840/commits/f5bd9e98da6d9f01e43299564d0f6922694cd90e

The latter one can be further renamed to `sync_ac`, but I think it's not as easy to understand as `sync_access_control`.